packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -412,20 +411,28 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   }, [uploads]);
 
   const onSendMessageHandler = useCallback(
-    async function (content: string, /* @conditional-compile-remove(attachment-upload) */ options?: MessageOptions) {
+    async function (
+      content: string,
+      /* @conditional-compile-remove(attachment-upload) */ messageOptions?: MessageOptions
+    ) {
       /* @conditional-compile-remove(attachment-upload) */
-      const attachments = options?.attachments ?? [];
+      const attachments = messageOptions?.attachments ?? [];
       /* @conditional-compile-remove(attachment-upload) */
       handleUploadAction({ type: AttachmentUploadActionType.Clear });
       /* @conditional-compile-remove(attachment-upload) */
       await adapter.sendMessage(content, {
-        attachments: attachments
+        attachments: attachments,
+        /* @conditional-compile-remove(rich-text-editor-composite-support) */
+        type: options?.richTextEditor ? 'html' : 'text'
```

Comment:
we probably need to use the value from `messageOptions` instead and update the place where we call `onSendMessageHandler` callback (in SendBox and RichTextSendBox) so it's called with the correct values